sky/shell/tracing_controller.cc

```diff
@@ -25,6 +27,8 @@ const char kObservatoryMethodStopTracing[] = "flutter_stopTracing";
 const char kObservatoryResultOk[] = "{\"success\" : true}";
 const char kObservatoryResultFail[] = "{\"success\" : false}";
 
+const char kTracingSwitchEnableOnStartup[] = "startup-trace";
```

Comment:
I probably would have called this "trace-startup":
https://code.google.com/p/chromium/codesearch#chromium/src/components/tracing/tracing_switches.cc&q=trace-startup&sq=package:chromium&type=cs&l=33